src/common/util/userCan.js

```diff
@@ -45,11 +47,7 @@ const CAPABILITY_ROLES = {
   monitorJobQueues: [ADMIN],
 }
 
-export const VALID_ROLES = Object.keys(CAPABILITY_ROLES).map(capability => {
-  return CAPABILITY_ROLES[capability]
-}).reduce((prev, curr) => {
-  return [...new Set(prev.concat(curr))]
-}, [])
+export const ROLES = [...GENERAL_USE]
```

Comment:
No need to export from here...wherever else we need these values, they can just be imported directly from `common/models/user`.